diagnostic.md

```diff
@@ -58,13 +62,25 @@ the above `Movies` and `Profiles`.
 1.  What is `Dependent: Destroy` and where/why would we use it?
 
 ```sh
-  # < Your Response Here >
+dependent: destroy would be used in the command line and you would use it to remove
+that object from the table.
 ```
```

Comment:
it's used in rails, and it ensures that when something's deleted, all associated objects also are.